ui/app/AppLayouts/Chat/popups/community/BackUpCommunityPopup.qml

```diff
@@ -0,0 +1,142 @@
+import QtQuick 2.12
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Core.Utils 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+import StatusQ.Popups 0.1
+
+import utils 1.0
+import shared.controls 1.0
+import shared 1.0
+
+StatusStackModal {
+    id: root
+
+    property var store
+    property string privateKey
+    property int maximumLength: 4
+
+    width: 480
+    height: 504
+    header.title: (currentIndex === 0) ? qsTr("Back up community key")
+                                       : qsTr("Confirm your community key")
+    padding: Style.current.padding
+
+    stackItems: [
+    ColumnLayout {
+        spacing: Style.current.halfPadding
+
+        StatusInput {
+            id: pKeyInput
+            Layout.fillWidth: true
+            leftPadding: 14
+            rightPadding: Style.current.halfPadding
+            topPadding: 0
+            bottomPadding: 0
+            label: qsTr("Your community key")
+            minimumHeight: 56
+            maximumHeight: 56
+            input.text: Utils.getElidedPk(root.privateKey)
+            input.edit.readOnly: true
+            input.rightComponent: StatusButton {
+                anchors.verticalCenter: parent.verticalCenter
+                borderColor: Theme.palette.primaryColor1
+                size: StatusBaseButton.Size.Tiny
+                text: qsTr("Copy")
+                onClicked: {
+                    text = qsTr("Copied");
+                    root.store.copyToClipboard(root.privateKey);
+                }
+            }
+        }
+
+        StatusBaseText {
+            Layout.fillWidth: true
+            Layout.preferredHeight: 36
+            Layout.topMargin: Style.current.halfPadding
+            text: qsTr("You should keep it safe and only share it with people you trust to take ownership of your community")
+            wrapMode: Text.WordWrap
+            font.pixelSize: 13
+            color: Theme.palette.baseColor1
+        }
+
+        StatusBaseText {
+            Layout.fillWidth: true
+            Layout.preferredHeight: 34
+            text: qsTr("You can also use this key to import your community on another device")
+            wrapMode: Text.WordWrap
+            font.pixelSize: 13
+            color: Theme.palette.baseColor1
+        }
+        },
+        Item {
```

Comment:
Looks like ColumnLayout uses 4 spaces on left and the next Item uses 8 spaces on left. In order this to look more consistent we need to use same number of spaces on left.

Reply:
@faust4exe are you referring to the column `spacing`?